xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceMarketDataService.java

```diff
@@ -102,15 +102,16 @@ private OrderBook getBinanceOrderBook(Instrument instrument, Object... args) thr
   }
 
   public static OrderBook convertOrderBook(BinanceOrderbook ob, Instrument pair) {
+    Date timeStamp = Date.from(Instant.now());
     List<LimitOrder> bids =
             ob.bids.entrySet().stream()
-                    .map(e -> new LimitOrder(OrderType.BID, e.getValue(), pair, null, null, e.getKey()))
+                    .map(e -> new LimitOrder(OrderType.BID, e.getValue(), pair, null, timeStamp, e.getKey()))
```

Comment:
this conflicts with this fields documentation:

a Date object representing the order's timestamp according to the exchange's server, null if not provided